extension/app/src/components/input_bar/InputBarCitations.tsx

```diff
@@ -26,31 +26,29 @@ export function InputBarCitations({
       const isImage = Boolean(blob.preview);
 
       nodes.push(
-        <>
-          <Citation
-            disabled={disabled}
-            key={`cf-${blob.id}`}
-            className="w-48"
-            isLoading={blob.isUploading}
-          >
-            {isImage ? (
-              <>
-                <CitationImage imgSrc={blob.preview ?? ""} />
-                <CitationIcons>
-                  <Icon visual={ImageIcon} />
-                </CitationIcons>
-              </>
-            ) : (
+        <Citation
```

Comment:
Unrelated, I just removed the fragment cause we're in a loop and the key is set on Citation so it was causing a React error.